server/tests/backends/sql_translator_unit_tests/test_base_translator.py

```diff
@@ -571,7 +580,7 @@ def test_text(base_translator: BaseTranslator, default_step_kwargs: dict[str, An
     ctx = base_translator.text(step=step, columns=selected_columns, **default_step_kwargs)
 
     expected_query = Query.from_(previous_step).select(
-        *selected_columns, ValueWrapper(text).as_(new_column_name)
+        *selected_columns, functions.Cast(ValueWrapper(text), "TEXT").as_(new_column_name)
```

Comment:
```suggestion
*selected_columns, functions.Cast(ValueWrapper(text), self.DATA_TYPE_MAPPING.text).as_(new_column_name)
```

not a blocking comment tho...

Reply:
It's a test, so I want as few variables as possible to be as precise as possible, in order to limit bugs in the test itself